app/views/spree/admin/gift_cards/edit.html.erb

```diff
@@ -67,15 +67,23 @@
           <%= f.text_field :send_email_at, :class => 'datepicker datepicker-from giftcard-datepicker' %>
           <% end %>
         <% end %>
+
+        <% if @gift_card.deactivated_at %>
+          <div class="deactivated-at">
+            Deactivated at: <%= @gift_card.formatted_deactivated_at %>
+          </div>
+        <% end %>
       </div>
     </div>
 
     <%= render :partial => 'spree/admin/shared/edit_resource_links', locals: { collection_url: edit_admin_order_path(@order) } %>
   </fieldset>
+<% end %>
+
 
-  <div class="form-buttons filter-actions actions" data-hook="buttons">
-    <% if @order.completed? %>
-      <%= button_link_to "Send email now", send_email_admin_order_gift_card_path(@order, @gift_card), :icon => 'email', :data => { :confirm => "Any changes that have not been updated will not send" } %>
-    <% end %>
+<% if @order.completed? && !@gift_card.deactivated? && @order.paid? %>
```

Comment:
oh good thinking, i wasn't even thinking that far ahead, just trying to push the logic to the model.